style/visual.less

```diff
@@ -44,6 +44,16 @@
                 shape-rendering: crispEdges;
             }
 
+            .nodeRect:focus {
+                outline: none;
+            }
+
+            .nodeRect:focus-visible {
```

Comment:
works on Edge, Chrome, Firefox?

Reply:
works as expected on Edge, Chrome and Firefox